<!DOCTYPE html>
<html>
<head>
<title>Wed Jun 23, 2021 : #batlab (chtc)</title>
</head>
<body>
<h3>Wed Jun 23, 2021 : #batlab (chtc)</h3>
<span style="color: #235e5b"><span style="font-size: small">(03:39:21)</span> <b>johnkn:</b></span> I submitted 6 test runs against this build <a href="https://batlab.chtc.wisc.edu/results/run-details.php?runid=546699">https://batlab.chtc.wisc.edu/results/run-details.php?runid=546699</a> overnight<br/>
<span style="color: #d58247"><span style="font-size: small">(10:15:40)</span> <b>gthain:</b></span> So, still a bunch of random failures, but not to the extent where the test suite falls over, as it was doing before the port exhaustion fixes<br/>
<span style="color: #235e5b"><span style="font-size: small">(11:18:21)</span> <b>johnkn:</b></span> new test run (load7) added to the same build run as the last load test.<br/>
<span style="color: #235e5b"><span style="font-size: small">(11:18:35)</span> <b>johnkn:</b></span> Should I do a bunch of test runs?<br/>
<span style="color: #235e5b"><span style="font-size: small">(13:12:04)</span> <b>johnkn:</b></span> Looks to me like significantly more test failures on the latest load test.<br/>
<span style="color: #235e5b"><span style="font-size: small">(13:12:29)</span> <b>johnkn:</b></span> and also two platforms show "interrupted"<br/>
<span style="color: #d58247"><span style="font-size: small">(15:30:05)</span> <b>gthain:</b></span> Another "Signal 8" build failure this afternoon<br/>
<span style="color: #d58247"><span style="font-size: small">(20:30:29)</span> <b>gthain:</b></span> Huh:<br/>
<span style="color: #d58247"><span style="font-size: small">(20:30:32)</span> <b>gthain:</b></span> <pre>$ condor_q -hold<br/><br/><br/>-- Schedd: <a href="http://batlabsubmit0001.chtc.wisc.edu">batlabsubmit0001.chtc.wisc.edu</a> : &lt;128.104.100.94:9618?... @ 06/23/21 20:29:02<br/> ID        OWNER          HELD_SINCE  HOLD_REASON<br/>675785.0   condorauto      6/23 16:46 Error from <a href="mailto:slot1_5@e420.chtc.wisc.edu">slot1_5@e420.chtc.wisc.edu</a>: runtime/cgo: pthread_create failed: Resource temporarily unavailable SIGABRT: abort PC=0x14bc649343d7 m=0 sigcode=18446744073709551610  goroutine 0 [idle]: runtime: unknown pc 0x14bc649343d7 stack: frame={sp:0x7ffe0a4b58d8, fp:0x0} stack=[0x7ffe0a2b6d48,0x7ffe0a4b5d80) 00007ffe0a4b57d8:  0000000000000000  0000000000000000  00007ffe0a4b57e8:  0000000000000000  0000000000000000  00007ffe0a4b57f8:  0000000000000000  0000000000000000  00007ffe0a4b5808:  0000000000000000  0000000000000000  00007ffe0a4b5818:  00<br/><br/>1 jobs; 0 completed, 0 removed, 0 idle, 0 running, 1 held, 0 suspended</pre><br/>
<blockquote>
<span style="color: #2b6836"><span style="font-size: small">(22:45:19)</span> <b>jcpatton:</b></span> A CHTC user was complaining about getting "Resource temporarily unavailable" in one of their jobs, but I never caught the hostname. Wonder if it's the same machine.<br/>
</blockquote>
</body>
</html>
